src/content/docs/validator/external-chains/xrpl.mdx

```diff
@@ -147,12 +147,26 @@ You can query the node with:
 curl -H 'Content-Type: application/json' -d '{"method":"server_info","params":[{}]}' http://localhost:5005
 ```
 
+## Recovering From Crashes
 
-### Ampd Configuration
-Once your node is up and running, you need to add the `XRPL` chain to your [ampd daemon's](https://github.com/axelarnetwork/axelar-amplifier/tree/main/ampd) `config.toml`file. This can be done by adding in the following configuration:
+Due to ongoing changes in the Cosmos SDK your Axelar node may run into issues when running (such as an `AppHash` error). An example of such an error may appear as follows:
```

Comment:
the issue isn't ongoing changes, but due to a bug in the underlying cosmos database that causes it to fail under load such that Axelar node gets an AppHash error